tasks/connectors/snyk_v2/lib/snyk_v2_client.rb

```diff
@@ -6,21 +6,20 @@ module SnykV2
       class SnykV2Client
         class ApiError < StandardError; end
 
-        HOST = "https://snyk.io"
-
-        def initialize(token)
+        def initialize(token, api_base_url)
           @token = token
+          @api_base_url = "https://#{api_base_url}/v1"
```

Comment:
having api_base_url being just the base in snyk_v2_task and referring to the whole constructed url in snyk_v2_client is confusing and makes the code harder to read, can we change this to be @host or just @api_url ?